programs/programs/policyengine/calculators/dependencies/spm.py

```diff
@@ -212,14 +212,25 @@ def value(self):
 
 class NcTanfCountableGrossUnearnedIncomeDependency(SpmUnit):
     field = "nc_tanf_countable_gross_unearned_income"
-    dependencies = (
-        "income_type",
-        "income_amount",
-        "income_frequency",
-    )
+    income_types = [
+        "pension",
+        "veteran",
+        "unemployment",
+        "sSDisability",
+        "workersComp",
+        "sSRetirement",
+        "deferredComp",
+        "rental",
+        "childSupport",
+        "alimony",
+        "investment",
+        "sSSurvivor",
+        "sSDependent",
+        "boarder",
+    ]
 
     def value(self):
-        return int(self.screen.calc_gross_income("yearly", ["unearned"]))
+        return self.screen.calc_gross_income("yearly", self.income_types)
```

Comment:
We have the `exclude functionality built in already.
```suggestion
        return self.screen.calc_gross_income("yearly", ["unearned"], exclude=["sSI"])
```